Rakefile

```diff
@@ -302,8 +324,33 @@ We use information from your github account throughout the class.
         verify: -> { assert(!github_username.to_s.empty?, "We can't find your github username.", "") }
       },
 
+      global_gitignore: {
+        header: "Register a global gitignore file",
+        installation_steps: [
+          %q(
+1. Backup your existing global_gitignore (if it exists).  You can ignore a "No such file or directory" error:
+
+    $ mv ~/.gitignore_global ~/.gitignore_global.bak
+
+2. Download the provided global gitignore file to your home dir:
+
+    $ curl -sSL https://raw.githubusercontent.com/ga-dc/installfest/master/support/gitignore_global -o ~/.gitignore_global
```

Comment:
Please review support/gitignore_global.  I feel it is too big to copy/paste (just asking for trouble).  We *can* shrink it down significantly by just using files known to be used in GA exercises.  I was impressed to find that someone was cultivating a reusable gitignore ignore file, so I tried to use it.